packages/control-property-editor-common/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@sap-ux-private/control-property-editor-common",
+    "displayName": "Control Property Editor",
```

Comment:
Shouldn't this be called something else?

Reply:
we had a proposal from Tobias for "@sap-ux-private/control-property-editor-api" it was left for preference so i left it as it is.

Reply:
I think @zdravko-georgiev talks about the `displayName`

Reply:
Yes, I mean the `displayName`. We have also the same display name for the Control Property Editor